docs/_cli_docs/cli.md

```diff
@@ -64,24 +64,22 @@ This command is typically followed by `zapier upload`.
 
 ## convert
 
-> Convert a Legacy Web Builder app or Visual Builder integration to a CLI integration.
+> Convert a Visual Builder integration to a CLI integration.
 
 **Usage**: `zapier convert INTEGRATIONID PATH`
 
-If you're converting a **Legacy Web Builder** app: the new integration will have a dependency named zapier-platform-legacy-scripting-runner, a shim used to simulate behaviors that are specific to Legacy Web Builder. There could be differences on how the shim simulates and how Legacy Web Builder actually behaves on some edge cases, especially you have custom scripting code.
-
-If you're converting a **Visual Builder** app, then it will be identical and ready to push and use immediately!
+The resulting CLI integraiton will be identical to its Visual Builder version and ready to push and use immediately!
```

Comment:
`integraiton` spelling

Reply:
Shoot; good catch, I'll update this here and open up a PR to fix the source: https://github.com/zapier/zapier-platform/pull/613